airflow/templates/deployments-scheduler.yaml

```diff
@@ -18,17 +18,17 @@ spec:
       restartPolicy: Always
       containers:
         - name: "{{- .Values.airflow.prefix -}}scheduler"
-          image: {{ .Values.airflow.image }}
+          image: {{ .Values.airflow.image }}:{{ .Values.airflow.imageTag }}
           imagePullPolicy: {{ .Values.airflow.image_pull_policy}}
-          envFrom:
-            - configMapRef:
-                name: "{{- .Values.airflow.prefix -}}airflow-env"
-          {{- if .Values.airflow.airflow_cfg.enable }}
-          volumeMounts:
-            - mountPath: /usr/local/airflow-cm/
-              name: "{{- .Values.airflow.prefix -}}airflow-cfg"
-          command: ["bash", "/usr/local/airflow-cm/bootstrap.sh"]
-          {{- end }}
+          env:
+            - name: POSTGRES_HOST
+              value: {{ template "airflow.postgresql.fullname" . }}
```

Comment:
this look like a regression to me, you loose all the configuration file flexibility